.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: php
 
-php: [7.0, 7.1]
+php: [7.1, 7.2]
```

Comment:
phpSpec supports ^7.1, so should this extension

Reply:
You can remove 7.1 as we use 7.2 since 2.0

Reply:
I agree with @pamil. We could very well update the dev-dependencies of Akeneo 2.x too, not just 2.3. I think we should let 7.1.

However, we can add 7.3 for the future, as it is now available on Travis (RC3 currently).